methods.py

```diff
@@ -1642,3 +1642,7 @@ def get_default_include_paths(env):
         print_warning("Failed to find the include paths in the compiler output.")
         return []
     return [x.strip() for x in match[1].strip().splitlines()]
+
+
+def sorted_unique_list(list_to_sort):
+    return sorted(list(set(list_to_sort)))
```

Comment:
I'm not sure it's warranted to add a helper method for this here if it's only used twice in a single file. I'd define it where it's needed, and it can be generalized later if there's a need for it in other places.